manifests/oracle-cluster/sdlc/helium/account-postgres-sink-service.yaml

```diff
@@ -25,6 +25,9 @@ data:
               "table": "mobile_hotspot_infos",
               "schema": "public"
             }
+          ],
+          "crons": [
+            { "schedule": "0 12 * * *", "type": "integrity-check" }
```

Comment:
What does the `integrity-check` do?

Reply:
nvm

Reply:
fetches the last 24hrs worth of transactions for the specified program. Pulls in all unique writable accounts off those transactions and updates them if a deep equal returns false. Then increments a integrity-check prometheus counter.

So we can see if the webhooks are updating properly